tests/unittests/test_currently_syncing.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+from unittest import mock
+from tap_github.sync import (update_currently_syncing_repo, update_currently_syncing,
+                             get_ordered_stream_list, get_ordered_repos)
+
+
+class TestGetOrderedStreamList(unittest.TestCase):
+    
+    def test_for_interrupted_sync(self):
```

Comment:
Add test comments in all the unit tests.

Reply:
Added docstrings and comments in unit tests